frontend/src/metabase/services.js

```diff
@@ -91,6 +92,8 @@ export async function runQuestionQuery(
     isDirty = false,
     ignoreCache = false,
     collectionPreview = false,
+    // Ability to override or add extra query params to the request, used by Embedding SDK
+    queryParamsOverride = {},
```

Comment:
This is very important to have and will be used by no-user embedding via SDK.
We have to pass parameters as a `parameters` fiekd with a JSON string containing params for the `/api/embed` endpoints:

```
queryResults = await runQuestionQuery(question, {
      cancelDeferred,
      ignoreCache: false,
      isDirty: isQueryDirty,
      ...(isStaticEmbedding && {
        queryParamsOverride: {
          parameters: JSON.stringify(filteredParameters),
        },
      }),
    });
```